README.md

```diff
@@ -1 +1,107 @@
+- Length = 0xXX (1 byte) - The sum of the number of bytes of the fields "Application Data" and "Checksum".
+- Application Data = 0xXX ... 0xXX (2-16 bytes) - The response with the data requested by the corresponding command.
+- Checksum = 0xXX (1 byte) - Used to check the integrity of the packet.  It is calculated by adding all bytes in "Command Code", "Length", and "Application Data" fields and extracting the least significant byte of the result.
+    - E.g. Successful calculation of angular position with angles alpha = 5 degrees (0x40 A0 00 00) and beta = 10 degrees (0x41 20 00 00)
+        - DE AD BE EF 01 0A 40 A0 00 00 41 20 00 00 00 
+        - 01 + 0A + 40 + A0 + 00 + 00 + 41 + 20 + 00 + 00 + 00 = 0x14C.  Least significant byte of 0x14C is 0x4C.
+        - So checksum is 0x4C
+
+### SPI Protocol Format
+The following describes an example of a command/response communication for command code 0x01 via SPI.  Note that the IDLE byte value is 0xFF:
+
+- SPI Request:
+    - On board computer to generic fine sun sensor device:
+        - Sync Word = 0xDEADBEEF (4 bytes)
+        - Command Code - 0x01 (1 byte)
+        - Length = 0x01 (1 byte)
+        - Checksum = 0x02 (1 byte)
+    - Generic fine sun sensor device to on board computer:
+        - Idle message = IDLE x 7 (7 bytes)
+- SPI Response:
+    - On board computer to generic fine sun sensor device:
+        - Idle message = IDLE x 16 (16 bytes)
+    - Generic fine sun sensor device to on board computer:
+        - Sync Word = 0xDEADBEEF (4 bytes)
+        - Command Code = 0x01 (1 byte)
+        - Length = 0x0A (1 byte)
+        - Application Data
+            - Angle alpha = 0xXX 0xXX 0xXX 0xXX (float, 4 bytes)
+            - Angle beta = 0xXX 0xXX 0xXX 0xXX (float, 4 bytes)
+            - Error code = 0xXX (1 byte)
+        - Checksum = 0xXX (1 byte)
+
+### Commands
+The command codes implemented are:
+- Command Code = 0x01, Name = Angular position, Functionality = Request for the angular position (alpha, beta) and the error code.
+
+#### Command 0x01:  Angular position
+Request for the angular position and corresponding error code.  The two angles which determine the angular position (alpha and beta, see above) are represented by a single-precision floating-point format.  The error code is represented in a char.
+- Command
+    - Sync Word = 0xDEADBEEF (4 bytes)
+    - Command Code = 0x01 (1 byte)
+    - Length = 0x01 (1 byte)
+    - Checksum = 0x02 (1 byte)
+
+- Response
+    - Sync Word = 0xDEADBEEF (4 bytes)
+    - Command Code = 0x01 (1 byte)
+    - Length = 0x0A (1 byte)
+    - Application Data:
+        - Angle alpha = 0xXX 0xXX 0xXX 0xXX (float, 4 bytes)
+        - Angle beta = 0xXX 0xXX 0xXX 0xXX (float, 4 bytes)
+        - Error code = 0xXX (1 byte)
+    - Checksum = 0xXX (1 byte)
+
+Error codes:
+- 0 = No error.  Angles were calculated successfully.
+- 1 = Error.  Sun not in field of view or earth albedo affected measurement.
```

Comment:
Need to include `Simulation` and `42` sections including how to configure those portions.

Reply:
Will do.